civihr_default_theme/scss/pages/_my-leave.scss

```diff
@@ -0,0 +1,10 @@
+.block-civihr-leave-absences {
```

Comment:
This is the class for the block, autogenerated by Drupal, so kept the namespace.